src/ImagePotentialBondGPU.cuh

```diff
@@ -0,0 +1,352 @@
+// Copyright (c) 2018-2020, Michael P. Howard
+// Copyright (c) 2021-2025, Auburn University
+// Part of azplugins, released under the BSD 3-Clause License.
+
+#include "hip/hip_runtime.h"
+#include "hoomd/HOOMDMath.h"
+#include "hoomd/Index1D.h"
+#include "hoomd/ParticleData.cuh"
+#include "hoomd/TextureTools.h"
+
+#include "hoomd/BondedGroupData.cuh"
+
+#include <assert.h>
+
+/*! \file PotentialBondGPU.cuh
+    \brief Defines templated GPU kernel code for calculating the bond forces.
+*/
+
+#ifndef __IMAGE_POTENTIAL_BOND_GPU_CUH__
+#define __IMAGE_POTENTIAL_BOND_GPU_CUH__
+
+namespace hoomd
+    {
+namespace azplugins
+    {
+namespace kernel
+    {
```

Comment:
azplugins uses a two-tier namespace for GPU code. The data structure and kernel driver function should go in `azplugins::gpu`. The corresponding kernel should go in `azplugins::gpu::kernel`. That way, you don't have to put `gpu` or `kernel` in the function names (e.g., `compute_bond_forces` can be safely used for both). Please update.